CandidateReranking.md

```diff
@@ -74,27 +72,27 @@ Entites returned from the SemanticSearchAPI.
 
 | Param    | Type | Description
 |----------|------|--------|
-| name | String | **Required**: Name of the keyword, e.g. "Java".
-| type | String | **Required**: Type of the keyword, e.g. "skill".
-| weight | Integer | **Required**: Weight of the keyword as an int.
+| name | String | **Optional**: Name of the keyword, e.g. "Java".
+| type | String | **Optional**: Type of the keyword, e.g. "skill".
```

Comment:
if this is an enumeration, should we UPPER_CAMEL_CASE it?

Reply:
@thomasmahoney I would prefer that yes but alas the service is already in use and  this is the way we set up the contract initially. If we had done that then it would have removed the need for me to create the the SemanticName methods to pull the enumerated value. I think I made a lot of little mistakes like this that made my life harder...